client/my-sites/site-settings/style.scss

```diff
@@ -68,7 +68,7 @@
 		font-size: 13px;
 		font-style: italic;
 		font-weight: 400;
-		color: $gray;
+		color: $gray-dark;
```

Comment:
As suggested in https://github.com/Automattic/wp-calypso/pull/11127#issuecomment-277001916 I've updated this to `$gray-text`, which is basically `darken( $gray, 18% );` or `#537994`.